server/helpers.js

```diff
@@ -119,3 +121,13 @@ export function isCredentialError(err) {
            err.errCode === getErrorCode('INVALID_PARAMETERS') ||
            err.errCode === getErrorCode('NO_PASSWORD');
 }
+
+export function setupMoment(locale) {
+    if (locale) {
```

Comment:
Hmm, setupTranslator is shared between client and server. But this should be done when we use moment (I plan to do so)